README.md

```diff
@@ -80,8 +80,8 @@ pytest
 
 # Run demo
 python src/demo.py --help
-# To output debug logs to a file, change DEBUG_LOG_RESPONSE to True in opower.py and run:
-python src/demo.py --verbose 2> out.txt
+# To output debug logs to a file run:
```

Comment:
`# To output debug logs and API responses to a file run:`

Reply:
Updated. I fixed diff and also updated logging for smud provider